code/modules/mob/living/simple_animal/abnormality/!nonstandard/scp/tickle_monster.dm

```diff
@@ -0,0 +1,83 @@
+/mob/living/simple_animal/hostile/abnormality/tickle_monster
+	name = "The Tickle Monster"
+	desc = "An orange blob that is always looking for a hug."
+	icon = 'ModularTegustation/Teguicons/32x32.dmi'
+	icon_state = "999"
+	icon_living = "999"
+	is_flying_animal = TRUE
+	maxHealth = 600	//It is helpful and therefore weak.
+	health = 600
+	//work stuff
+	can_breach = TRUE
+	start_qliphoth = 4
+	threat_level = ZAYIN_LEVEL
+	work_damage_amount = 4
+	work_damage_type = WHITE_DAMAGE
+
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 1, WHITE_DAMAGE = 1, BLACK_DAMAGE = 1.5, PALE_DAMAGE = 2)
+	melee_damage_lower = 0
+	melee_damage_upper = 0
+	melee_damage_type = WHITE_DAMAGE
+	faction = list("neutral", "hostile")	//Mostly Just vibes
+	work_chances = list(
+						ABNORMALITY_WORK_INSTINCT = list(90, 80, 75, 60, 60),
+						ABNORMALITY_WORK_INSIGHT = list(40, 40, 40, 40, 40),
+						ABNORMALITY_WORK_ATTACHMENT = list(90, 80, 75, 60, 60),
+						ABNORMALITY_WORK_REPRESSION = list(40, 40, 40, 40, 40),
+						"Release" = 0
+						)
+	ego_list = list(
+//		/datum/ego_datum/weapon/hugs,
+		/datum/ego_datum/armor/hugs
+		)
+//	gift_type = /datum/ego_gifts/assonance
+	abnormality_origin = ABNORMALITY_ORIGIN_SCP
+
+	//slowly heals sanity over time
+	var/heal_cooldown
+	var/heal_cooldown_time = 3 SECONDS
+	var/heal_amount = 5
+
+
+/mob/living/simple_animal/hostile/abnormality/tickle_monster/Life()
+	. = ..()
+	if(!.) // Dead
+		return FALSE
+	if((heal_cooldown < world.time) && !(status_flags & GODMODE))
+		HealPulse()
+
+/mob/living/simple_animal/hostile/abnormality/tickle_monster/Initialize()
+	. = ..()
+	RegisterSignal(SSdcs, COMSIG_GLOB_MOB_DEATH, .proc/on_mob_death) // Hell
+
+/mob/living/simple_animal/hostile/abnormality/tickle_monster/Destroy()
+	UnregisterSignal(SSdcs, COMSIG_GLOB_MOB_DEATH)
+	return ..()
```

Comment:
Could probably make it have less health and add panics to this.

Reply:
Wouldn't this require a new signal?